roles/dellhw_exporter/defaults/main.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2025 VEXXHOST, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+dellhw_exporter_helm_repository_name: prometheus-dellhw-exporter
+dellhw_exporter_helm_repository_url: https://galexrt.github.io/dellhw_exporter
+dellhw_exporter_helm_chart_name: prometheus-dellhw-exporter
+dellhw_exporter_helm_chart_version: 1.0.1
+dellhw_exporter_helm_app_chart_version: 2.0.0-rc.1
```

Comment:
I feel like we should drop this, since this is more of what version we're running as opposed to the helm app chart version, I think we've done a similar `_version` in other roles.